Description: Do not install manual via Makefile.
Author: Ermakov Alexander <ermakovav@altgtu.ru>

---
--- dupd-1.7.3.orig/Makefile
+++ dupd-1.7.3/Makefile
@@ -43,7 +43,6 @@ endif
 # https://www.gnu.org/prep/standards/html_node/DESTDIR.html
 
 INSTALL_PREFIX ?= /usr/local
-MAN_BASE ?= $(INSTALL_PREFIX)/man
 
 BUILD=$(TOP)/build
 CCC=$(CC) -Wall -Wextra -std=gnu99 $(OPT)
@@ -196,9 +195,6 @@ release:
 install: dupd
 	mkdir -p $(DESTDIR)$(INSTALL_PREFIX)/bin/
 	cp dupd $(DESTDIR)$(INSTALL_PREFIX)/bin/
-	mkdir -p $(DESTDIR)$(MAN_BASE)/man1
-	cp man/dupd.1 $(DESTDIR)$(MAN_BASE)/man1
 
 uninstall:
 	rm -f $(DESTDIR)$(INSTALL_PREFIX)/bin/dupd
-	rm -f $(DESTDIR)$(MAN_BASE)/man1/dupd.1
