Description: Fix path to binary file for autopkgtest
Author: Ermakov Alexander <ermakovav@altgtu.ru>
Forwarded: not-needed
Last-Update: 2026-02-25

--- dupd-1.7.3.orig/tests/run
+++ dupd-1.7.3/tests/run
@@ -30,9 +30,9 @@
 if ($ENV{'DUPD_VALGRIND'}) {
     $ENV{'DUPD_CMD'} =
         "valgrind --error-exitcode=1 --tool=memcheck " .
-        "--leak-check=full --show-reachable=yes ../dupd";
+        "--leak-check=full --show-reachable=yes dupd";
 } else {
-    $ENV{'DUPD_CMD'} = "../dupd";
+    $ENV{'DUPD_CMD'} = "dupd";
 }
 
 # Needed for sort to work correctly
